lib/node_modules/@stdlib/stats/base/dists/geometric/logpmf/examples/c/example.c

```diff
@@ -17,7 +17,6 @@
 */
 
 #include "stdlib/stats/base/dists/geometric/logpmf.h"
-#include "stdlib/math/base/special/round.h"
```

Comment:
This should not be removed as its used in L36/L

Notice that the comment was for the line with `#include <math.h>`. You want to double-check which lines need to be updated per the comment.